axonserver/src/main/java/io/axoniq/axonserver/localstorage/TrackingEventProcessorManager.java

```diff
@@ -177,11 +180,16 @@ public void close() {
         private final StreamObserver<InputStream> eventStream;
         private volatile CloseableIterator<SerializedEventWithToken> eventIterator;
         private volatile boolean running = true;
+        private final Set<PayloadDescription> blacklistedTypes = new CopyOnWriteArraySet<>();
+        private volatile int force = blacklistedSendAfter;
```

Comment:
I am aware what `force` if for, but I don't think the name is overly descriptive.
Can we rename it to something else, for example `forceEventHandling`?